config/tailwind.js

```diff
@@ -229,7 +229,7 @@ module.exports = {
       '3xl': '48rem',
       '4xl': '56rem',
       '5xl': '64rem',
-      '6xl': '72rem',
+      '6xl': '73.1429rem', // 1024px
```

Comment:
6xl wasn't being used yet, so should be fine to change. This is used for the max width of the content within a section